frontend/src/pages/Profile/ConnectedProfile.scss

```diff
@@ -0,0 +1,72 @@
+@import "../../variable.scss";
+
+@mixin style1($color, $width, $height, $font-size, $margin-bottom) {
+  background: $color;
+  width: $width;
+  height: $height;
+  font-size: $font-size;
+  margin-bottom: $margin-bottom;
+}
+
+@mixin style2($width, $heigth, $margin-top) {
+  width: $width;
+  height: $heigth;
+  margin-top: $margin-top;
+}
+
+* {
+  font-family: $fontFamily;
+}
+
+main {
+  height: 100vh;
+
+  .headband {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 25vh;
+    width: 100vw;
+    box-shadow: $boxShadow;
+    border-bottom: $borderDark;
+    position: relative;
+    z-index: 10;
+
+    .profilePict {
+      height: 20vh;
+      margin: 1rem;
```

Comment:
put this here to fit figma style : filter: drop-shadow(0px 0px 3px black);

Reply:
Thanks and edited